src/many-ledger/src/main.rs

```diff
@@ -45,6 +45,10 @@ struct Opts {
     #[clap(long)]
     persistent: PathBuf,
 
+    /// Path to snapshot
+    #[clap(long)]
+    snapshots: PathBuf,
```

Comment:
How hard would it be not to have snapshot functionality (error out on ABCI calls) if this argument is optional?